.github/pull_request_template.md

```diff
@@ -12,9 +12,9 @@ You will be automatically asked to sign the contributors license agreement (CLA)
 ##### Checklist
 
 - [ ] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-_____
-- [ ] Required: The PR title must be prefixed with a JIRA Issue number. For example: "ICU-1234 Fix xyz"
+- [ ] Required: The PR title must be prefixed with a JIRA Issue number. <!-- For example: "ICU-1234 Fix xyz" -->
```

Comment:
I guess that works. Or maybe put the sample prefix into `backticks`? But you want to hide it when not in edit mode?

Reply:
Thanks! Good idea to use backticks -- that would let the example still show up when not in edit mode.

I was wondering if we'd maybe only want the example to show in edit mode perhaps (?).
Though I don't really have a strong feeling either way though. 

I can update to use backticks if you think that's better. :)

Reply:
I am undecided. Up to you. :-)

Reply:
I think I'll go with only showing the example on the edit tab, as the checklist is already looking a bit long. :)